cmd/oras/internal/display/metadata/table/discover.go

```diff
@@ -55,21 +55,30 @@ func (h *discoverHandler) OnDiscovered(referrer, subject ocispec.Descriptor) err
 }
 
 // Render implements metadata.DiscoverHandler.
-func (h *discoverHandler) Render() error {
+func (h *discoverHandler) Render() (err error) {
 	if n := len(h.referrers); n > 1 {
-		fmt.Fprintln(h.out, "Discovered", n, "artifacts referencing", h.rawReference)
+		_, err = fmt.Fprintln(h.out, "Discovered", n, "artifacts referencing", h.rawReference)
```

Comment:
Wondering if the message can be simplified to something like "artifact(s)" so that we don't need the if-else clause🤔

Reply:
Someone thought this was important sometime. I think a topic of another PR though, this is just for lint